python/base-gpu/3.11/3.11.requirements.in

```diff
@@ -0,0 +1,2 @@
+-r requirements.in
```

Comment:
If these are in the `3.11` directory, do we still need the `3.11.` prefix on the file?

Reply:
Depends - if we want to have an identifier-level `requirements.in` that applies to all versions, we would need a separate version-specific `.in` file that's used to lock the requirements.txt for that Python version. The other solution is to copy/paste the same `requirements.in` between build directories, but that can become difficult to manage. I'm open to other ideas here though.

Reply:
After some additional testing, it turns out we're running into dependency resolution issues if we have to pin a different version in the `3.x.requirements.in` as opposed to the higher-level `requirements.in` -- need to follow up on this in a different PR, may just need to set different versions in different build directories and remove the package from the top level `requirements.in`.

Reply:
+1, I don't know some of the inner workings of python's dependency stuff, so I'll default to your initial layout.